wgengine/monitor/monitor_freebsd.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package monitor
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"net"
+	"strings"
+)
+
+type devdConn struct {
+	conn net.Conn
+}
+
+func NewConn() (Conn, error) {
+	conn, err := net.Dial("unixpacket", "/var/run/devd.seqpacket.pipe")
+	if err != nil {
+		log.Fatal("Dial error", err)
```

Comment:
Return an error, don't log.Fatal from here.

(this is another thing we're bad at in this codebase right now, but libraries shouldn't fatally exit)

Reply:
Totally agree... I missed it when porting over some proof of concept code.  I've made the change.